apps/accounts/automations/gather_account/manager.py

```diff
@@ -116,9 +116,10 @@ def _analyse_datetime_changed(self, ori_account, d, asset, username):
             if not date:
                 continue
 
-            pre_date = ori_account and getattr(ori_account, field)
-            if pre_date == date:
-                continue
+            # 服务器收集的时间和数据库时间一致，不进行比较，无法检测风险 不太对，先注释
+            # pre_date = ori_account and getattr(ori_account, field)
+            # if pre_date == date:
+            #     continue
 
             if date and date < timezone.now() - delta:
                 risks.append(
```

Comment:
There is no need to make any changes to the code as it appears to function correctly. The code checks whether datetime fields in an object have changed since a specific point in time and returns all objects that do. If you are looking for improvements, please provide more details on what aspects of correctness or performance would be most beneficial to optimize.